### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#495] [Esami & Esoneri M-Z]
### Traccia

<img width="868" alt="image" src="https://github.com/user-attachments/assets/d6150aa7-6a68-4824-bb5d-600e4509c998">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/b486c509-ff05-4864-bcbe-9a43d47b3931)
